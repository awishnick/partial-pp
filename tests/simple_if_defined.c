// RUN: -DDEFINED -UUNDEFINED
#if defined(DEFINED)
// this is defined
#endif

#if defined(UNDEFINED)
// this is undefined
#endif

#if defined(FOO)
// this is unknown and will be left
#endif
