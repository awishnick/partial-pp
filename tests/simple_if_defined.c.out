// RUN: -DDEFINED -UUNDEFINED
// this is defined


#if defined(FOO)
// this is unknown and will be left
#endif
